Source/Fuse.Controls.ScrollView/Triggers/WhileVisibleInScrollView.uno

```diff
@@ -6,6 +6,21 @@ using Fuse.Controls;
 namespace Fuse.Triggers
 {
 	/**
+		How the bounds of an element are treated.
+	*/
+	public enum WhileVisibleInScrollViewHow
+	{
+		/**
+			Activates the `WhileVisibleInScrollView` trigger when at least some of the element bounds are in view.
```

Comment:
Rephrase, "Activates while the element is at least partially within the ScrollView"